oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/DatabaseSlowStatement.java

```diff
@@ -21,16 +21,26 @@
 import lombok.Getter;
 import lombok.Setter;
 import org.apache.skywalking.oap.server.core.Const;
+import org.apache.skywalking.oap.server.core.analysis.IDManager;
+import org.apache.skywalking.oap.server.core.analysis.NodeType;
+import org.apache.skywalking.oap.server.core.config.NamingControl;
 
 import static org.apache.skywalking.oap.server.core.source.DefaultScopeDefine.DATABASE_SLOW_STATEMENT;
 
 @ScopeDeclaration(id = DATABASE_SLOW_STATEMENT, name = "DatabaseSlowStatement")
 public class DatabaseSlowStatement extends Source {
+    @Setter
+    private NamingControl namingControl;
```

Comment:
Take a look at `SourceBuilder`, how we initialize this there.

Reply:
Actually I'd like to do like `SourceBuilder`, but `DatabaseSlowStatement` must have the default construstor, just referring to `DispatcherManager.addIfAsSourceDispatcher` line#97.
Another way is I declare two constructors in `DatabaseSlowStatement`, i.e. the default and with `NamingControl`, what do you think of that?

Reply:
I see, how about adding a `DatabaseSlowStatementBuilder`?

Reply:
That's all right. Done!